docker/metrics_collector/metrics_collector.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+import os
+from time import sleep
+
+from prometheus_client import start_http_server, Gauge
+
+import logging
+import sys
+
+# de-register default-collectors
+from prometheus_client import REGISTRY, PROCESS_COLLECTOR, PLATFORM_COLLECTOR
+REGISTRY.unregister(PROCESS_COLLECTOR)
+REGISTRY.unregister(PLATFORM_COLLECTOR)
+
+# remove python-gargage-collectior metrics
+REGISTRY.unregister(REGISTRY._names_to_collectors['python_gc_objects_uncollectable_total'])
+#REGISTRY.unregister(REGISTRY._names_to_collectors['python_gc_objects_collected_total'])
+
+WIS2BOX_LOGGING_LOGLEVEL = os.environ.get('WIS2BOX_LOGGING_LOGLEVEL')
+# gotta log to stdout so docker logs sees the python-logs
+logging.basicConfig(stream=sys.stdout)
+# create our own logger using same log-level as wis2box
+logger = logging.getLogger('metrics_collector')
+logger.setLevel(WIS2BOX_LOGGING_LOGLEVEL)
+
+INTERRUPT = False
+
+def collect_file_data_by_topic(root,filematch):
+    """
+    helper-function to collect data using labels 'topic' from given root-path and file-type
```

Comment:
Add function annotation and docstring (example: https://github.com/wmo-im/wis2box/blob/main/wis2box/data/__init__.py#L67-L74)